tests/ops/op_math/test_controlled.py

```diff
@@ -1426,7 +1425,7 @@ def test_multi_ctrl():
     op = tape.operations[0]
     assert isinstance(op, Controlled)
     new_tape = tape.expand(depth=1)
-    assert qml.equal(new_tape[0], qml.Toffoli(wires=[3, 7, 0]))
+    assert equal_if_decomposed(new_tape[0], qml.Toffoli(wires=[3, 7, 0]))
```

Comment:
Again, if the expected output has changed, we should compare against the new expected output.